stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss

```diff
@@ -7,10 +7,13 @@
   box-sizing: border-box;
   display: none;
   min-height: 37.5rem;
-  padding: $spacing-64 $spacing-24 0 $spacing-48;
+  padding: $spacing-64 0 0 $spacing-48;
   position: absolute;
   top: 7.125rem;
-  z-index: 2;
+  z-index: 1;
+  max-width: $width-1440;
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
@Ashish-srij can you use mixin for transform

Reply:
@surbhi90 Done.